services/QuillLMS/app/controllers/teachers/progress_reports/concepts/concepts_controller.rb

```diff
@@ -13,11 +13,8 @@ def index
   end
 
   private def json_payload
-    concepts_json = current_user.all_classrooms_cache(key: 'teachers.progress_reports.concepts.concepts') do
-      concepts_as_json
-    end
     {
-      concepts: concepts_json,
+      concepts: concepts_as_json,
```

Comment:
Note: concepts_as_json is a little confusing. It should be called concepts_as_json_for_user. I will adjust, after review.